server/terraform/modules/myhealth/main.tf

```diff
@@ -190,6 +190,55 @@ resource "google_compute_region_network_endpoint_group" "neg" {
   depends_on = [google_project_service.service]
 }
 
+
+# Google Cloud Armor security policy.
+resource "google_compute_security_policy" "policy" {
+  name = "lb-security-policy"
+
+  # IP Block list. 
+  # Up to ten IP ranges can be created per rule.
+  # Each rule is identified by its priority, from 1=highest to maxint32=default.
+  rule {
+    action   = "deny(403)"
+    priority = "1000"
+    match {
+      versioned_expr = "SRC_IPS_V1"
+      config {
+        # Using an unused portion of the loopback block as a placeholder.
+        # Remove it once a real set of IP Ranges are present.
+        src_ip_ranges = ["127.0.0.255/32", "127.0.0.254"]
+      }
+    }
+    description = "Deny access to specified IPs"
+  }
+
+  # Every security policy must re-iterate the default rule at maxint32.
+  # The 'action' may be changed but not the other properties.
```

Comment:
Should we have the policy that this rule is never changed? I'm thinking about avoiding mistakes in the future if someone doesn't understand what they're doing. Main point is that only the `src_ip_ranges` should be edited in the block list.

Reply:
I could change the comment to effectively say do not touch, which is the right setting for this project.

Medium term, I think the entire IP blocking config will move into its own config. It is in a sense 'individual remote client' level data (a list of probable malicious user addresses) and not 'production system' level data. So running a complete terraform apply will... cause issues if the two lists ever get out of sync. (E.g. need to update server and accidentally clobber block list; or need to update block list but not bring server config to latest.)

But for now it's probably simplest to keep it here.